src/precompiles/secp256k1.rs

```diff
@@ -145,38 +155,46 @@ mod tests {
             hex::decode("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
                 .unwrap();
 
-        let res = ECRecover::run(&input, 3_000, &new_context()).unwrap().1;
+        let res = ECRecover::run(&input, 3_000, &new_context())
+            .unwrap()
+            .output;
         assert_eq!(res, expected);
 
         let input = hex::decode("47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad000000000000000000000000000000000000000000000000000000000000001b000000000000000000000000000000000000000000000000000000000000001b0000000000000000000000000000000000000000000000000000000000000000").unwrap();
         let expected =
             hex::decode("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
                 .unwrap();
 
-        let res = ECRecover::run(&input, 3_000, &new_context()).unwrap().1;
+        let res = ECRecover::run(&input, 3_000, &new_context())
+            .unwrap()
+            .output;
         assert_eq!(res, expected);
 
         let input = hex::decode("47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad000000000000000000000000000000000000000000000000000000000000001b0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001b").unwrap();
         let expected =
             hex::decode("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
                 .unwrap();
 
-        let res = ECRecover::run(&input, 3_000, &new_context()).unwrap().1;
+        let res = ECRecover::run(&input, 3_000, &new_context())
+            .unwrap()
+            .output;
         assert_eq!(res, expected);
 
         let input = hex::decode("47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad000000000000000000000000000000000000000000000000000000000000001bffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff000000000000000000000000000000000000000000000000000000000000001b").unwrap();
         let expected =
             hex::decode("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")
                 .unwrap();
 
-        let res = ECRecover::run(&input, 3_000, &new_context()).unwrap().1;
+        let res = ECRecover::run(&input, 3_000, &new_context())
+            .unwrap()
+            .output;
         assert_eq!(res, expected);
 
         // Why is this test returning an address???
         // let input = hex::decode("47173285a8d7341e5e972fc677286384f802f8ef42a5ec5f03bbfa254cb01fad000000000000000000000000000000000000000000000000000000000000001b000000000000000000000000000000000000000000000000000000000000001bffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff").unwrap();
         // let expected = hex::decode("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff").unwrap();
         //
-        // let res = ecrecover_raw(&input, Some(500)).unwrap().1;
+        // let res = ecrecover_raw(&input, Some(500)).unwrap().output;
```

Comment:
Lol, updating the commented out code is very thorough! But maybe we don't need this block of commented out code any more?

Reply:
`ctrl + R` and going through them one by one to make sure it changed what you want :P, but anyways, a similar commented out code was in another library, and there potentially is a quirk here that shouldn't be happening from what I recall. From looking at the implementation though, it should still be compatible.